azurerm/internal/services/compute/windows_virtual_machine_scale_set_resource.go

```diff
@@ -918,6 +955,8 @@ func resourceArmWindowsVirtualMachineScaleSetRead(d *schema.ResourceData, meta i
 			d.Set("source_image_id", storageImageId)
 		}
 
+		enableAutomaticUpdates := true // default value of `enable_automatic_updates` to avoid unexpected diff
+		provisionVMAgent := true // default value of `provision_vm_agent` to avoid unexpected diff
```

Comment:
(as above) we cannot assume these are enabled by default - there's false-positives here - we need to default these to false and parse them out of the API